### PR TITLE
Snackbar and Grid Component Added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Snackbar 
Screenshot (12)

### Grid

- Simple Grid (two items in a grid)
- Simple Grid (three items in a grid)
- Simple Grid With card
Screenshot (13)
Screenshot (14)

### CODE REVIEW

**Review by sakleprajwal — APPROVED** [review]
Everything seems good. Please go ahead

File: JS/snackbar.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const snackbarExample = document.querySelector(".snackbar-bottom");
+const btnToShowSnackbar = document.querySelector(".snackbar-show-btn");
+const btnToCloseSnacknbar = document.querySelector(".snackbar-close-btn");
+
+let timerIdForToast;
+btnToShowSnackbar.addEventListener("click", ()=>{
+    snackbarExample.style.display = "flex";
+    timerIdForSnackbar = setTimeout(()=>{
+        snackbarExample.style.display = "none";
```
Comment (vaibhav18matere) [inline JS/snackbar.js:9]: could have used better naming instead of `snackbarExample`

File: component/grid.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    width: fit-content;
+    height: auto;
+}
+
+.grid-card{
+    width: 50%;
+    display: grid;
+    grid-template-columns: auto auto;
+    padding: 10px;
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 10px 36px 0px, rgba(0, 0, 0, 0.06) 0px 0px 0px 1px;
```
Comment (vaibhav18matere) [inline component/grid.css:30]: can make default class for `box-shadow` so you can use it in multiple places

**Review by vaibhav18matere — APPROVED** [review]
good work, just have small suggestions:

File: JS/snackbar.js (hunk 1 above)
Comment (Mahi24-prog) [reply JS/snackbar.js:9]: ok

File: component/grid.css (hunk 2 above)
Comment (Mahi24-prog) [reply component/grid.css:30]: ok

**Pull request merged**